### PR TITLE
feat: Add new throttle corruption

### PR DESCRIPTION
**Pull request by VolcanoCookies**
* Add new throttling corruption for media segments. `throttle={sq: *, rate: 1000}`
  * Throttling is specified in bytes per second.
  * Segment is kept in memory on the proxy and fed in chunks back to the client.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ To try it out, go to your favourite HLS/MPEG-DASH video player such as `https://
 | `/api/v2/manifests/dash/proxy-master.mpd` | GET    | Returns a proxy MPD file, based on query parameters                                                      |
 | `/api/v2/manifests/dash/proxy-segment`    | GET    | Applies corruption present in query parameter and may return a 302 redirect to the original segment file |
 | `/api/v2/segments/proxy-segment`          | GET    | Applies corruption present in query parameter and may return a 302 redirect to the original segment file |
+| `/api/v2/throttle`                        | GET    | Proxies a http request, throttling the response to a specified byte rate                                 |
```
Comment (birme) [inline README.md:57]: I'm thinking perhaps have the endpoint `/api/v2/segments/throttle` or what you think?

Comment (VolcanoCookies) [reply README.md:57]: If we only ever intend to throttle segments then yes that would be more descriptive. Through right now the throttling endpoint could be used to throttle any kind of request, so it depends on if we could see a reason to throttle manifest requests in the future or not.

Comment (birme) [reply README.md:57]: Good point, let's leave it as it is

**Review by birme — APPROVED** [review]
LGTM

**Pull request merged**